### ISSUE TITLE
Integrations test fails with Go 1.13beta1 (on 1.7 branch)

### ISSUE THREAD

**Issue by gottwald** [containous/traefik#5140]
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [X] No

### What did you do?

Use `FROM golang:1.13-rc-alpine` in the `build.Dockerfile` and run integration tests with `make test-integration`

### What did you expect to see?

Passing integration tests.

### What did you see instead?

This error:

```

----------------------------------------------------------------------
FAIL: https_test.go:333: HTTPSSuite.TestWithClientCertificateAuthenticationMultipeCAs

https_test.go:352:
    c.Assert(err, checker.NotNil, check.Commentf("should not be allowed to connect to server"))
... value = nil
... should not be allowed to connect to server


----------------------------------------------------------------------
MISS: https_test.go:395: HTTPSSuite.TestWithClientCertificateAuthenticationMultipeCAsMultipleFiles
OOPS: 0 passed, 1 FAILED, 1 MISSED
--- FAIL: Test (0.22s)
FAIL
exit status 1
FAIL	github.com/containous/traefik/integration	0.251s
make: *** [Makefile:75: test-integration] Error 1
```

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
n/a
Branch 1.7
Commit: 7db41967
plus the mentioned Dockerfile change above
```

Most probably related to https://github.com/golang/go/issues/32202
I'm raising the issue because the changed default behavior is TLS 1.3 protocol related and it looks like it's here to stay.

**Comment by dduportal** [comment]
Hi @gottwald , as Traefik is going to hit the [2.0 version soon](https://community.containo.us/t/traefik-release-v2-0-0-beta1/951), the version `v1.7` is not accepting new features, with the upcoming maintenance mode.

TLS 1.3 is supported on the `v2.0` , you can read more about this here: https://docs.traefik.io/v2.0/https/tls/ .

Thanks for this proposal, but we're gonna have to decline it, as TLS 1.3 won't be supported on `v1.7`.

**Comment by gottwald** [comment]
Sorry for the misunderstanding. I'm actually not proposing TLS 1.3 as a feature.
I was using Go 1.13 for a different reason in trying to fix a TLS Server Name problem we're hitting in production. As Go 1.13 ships an easy method to clone a http.Transport safely...

The problem I see and the reason why I raised this issue here is rather that this locks down a proper runtime update path for traefik 1.7 users.
Due to breaking changes in traefik 2.0 not all users will be able to switch right away. With no support for Go 1.13 they suddenly have a ticking clock, which is the sunset of Go 1.12 when 1.14 is released in February.
Having just two quarters or less (since traefik 2.0 is not released yet) to adapt to breaking changes is probably not well received by users with bigger setups.

**Comment by dduportal** [comment]
Thanks @gottwald for this clear explanation of the need. We have triaged this issue so it means that what I said has changed: we might want to support Go 1.13 for Traefik v1.7.X in the future. Thanks a lot for your interest!

**Comment by traefiker** [comment]
Closed by #5555.
